packages/app-admin/src/components/FileManager/FileManagerView.tsx

```diff
@@ -106,6 +107,21 @@ type FileManagerViewProps = {
     multipleMaxSize: number | string;
 };
 
+const graphql = {
```

Comment:
Please move this into the "./graphql", in the same fashion we already did with `LIST_FILES` and `CREATE_FILE`. Note that const naming (all uppercase) as well.